### PR TITLE
framework/task_manager : Add invalid return check for handle

### PR DESCRIPTION
**Pull request by jeongchanKim**
taskmgr_get_handle_by_pid can return negative value(-4, TM_UNREGISTERED_APP), but it is used for index of array.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [6924e143ebbfa06185b1d7811f7378dbd2833c43] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [6924e143ebbfa06185b1d7811f7378dbd2833c43] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [6e9f6340e3c3090dedb6bad99f6054854819ec5f] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [6e9f6340e3c3090dedb6bad99f6054854819ec5f] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [cec7c24d9d1c3cacf6267a52c5bea925274ba122] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [cec7c24d9d1c3cacf6267a52c5bea925274ba122] - Code Rule Check OK.

File: os/kernel/wdog/wd_cancel.c [hunk 1]
```diff
@@ -142,6 +142,9 @@ int wd_cancel(WDOG_ID wdog)
 		 */
 
 		ASSERT(curr);
+		if (!curr) {
```
Comment (sunghan-chang) [inline os/kernel/wdog/wd_cancel.c]: ASSERT(curr) is checking null for curr. redundant

**Comment by seinfra** [comment]
Target : [6e9f6340e3c3090dedb6bad99f6054854819ec5f] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [6b9a1c2762bea21e55b25834fce8971bf6bac85a] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [6b9a1c2762bea21e55b25834fce8971bf6bac85a] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [f23960c1b9849e929fd07b4817702830b20609de] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [f23960c1b9849e929fd07b4817702830b20609de] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [30ee4827da4b06c8bf4c7befaec4283686b38098] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [30ee4827da4b06c8bf4c7befaec4283686b38098] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [7dfa43f5354b2b0e52c96f95c3599c3ac7e2f1a5] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [7dfa43f5354b2b0e52c96f95c3599c3ac7e2f1a5] - Code Rule Check (C++) OK.

**Pull request merged**